### ISSUE TITLE
Cannot find module @babel/runtime/helpers/interopRequireWildcard.js

### ISSUE THREAD

**Issue by trilogo-lordee** [JeffreyWay/laravel-mix#2851]
-   Laravel Mix Version: 6.0.11 (`npm list --depth=0`)
``` 
├── axios@0.21.1
├── laravel-mix@6.0.11
├── lodash@4.17.20
└── postcss@8.2.6
```
-   Node Version (`v14.15.5`):
-   NPM Version (`6.14.11`):
-   OS: mint 20.1

### Description: 
After clear install laravel get error.

```
ERROR in ./resources/js/app.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Cannot find module '/home/kostia/Desktop/Server/Laragon/websocket/node_modules/@babel/runtime/helpers/interopRequireWildcard.js'
```

### Steps To Reproduce:
```
composer create-project laravel/laravel test
cd test
composer install
npm install
npm run dev
```

**Comment by trilogo-lordee** [comment]
I see that the package @babel/runtime:7.12.17 is missing some components.
And in package@babel/runtime:7.12.13 everything is good.

**Comment by jarichypma** [comment]
Same issue here, running with these specs:

```
├── axios@0.21.1
├── laravel-mix@6.0.11
├── lodash@4.17.20
├── postcss@8.2.6
├── resolve-url-loader@3.1.2
├── sass-loader@8.0.2
└── sass@1.32.7
```

- Node Version (v15.8.0):
- NPM Version (7.5.0):
- OS: MacOS 11.2.1 (M1)

### Description
```
ltest8 >> npx mix
ERROR in ./resources/js/app.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Cannot find module '/Users/mymacbook/code/php/ltest8/node_modules/@babel/runtime/helpers/interopRequireWildcard.js'
```

### Steps To Reproduce

Same as @trilogo-lordee described

**Comment by thecrypticace** [comment]
Not a Mix problem but a babel issue (see: https://github.com/babel/babel/issues/12816). It's already been fixed and tagged as v7.12.18. Please upgrade your deps (`yarn upgrade` or `npm upgrade`)
